forms-flow-web/src/components/App.jsx

```diff
@@ -14,8 +14,15 @@ import {
 const queryClient = new QueryClient();
 const App = React.memo((props) => {
   const { store, history, publish, subscribe, getKcInstance } = props;
+  const [isPreviewRoute,setIsPreviewRoute] = useState(false);
+  useEffect(()=> {
+    const location = window.location.pathname;
+    if(location.includes("view-edit")){
+      setIsPreviewRoute(true);
+    }
```

Comment:
pls change to set & import from Service MF helper function

Reply:
updated.